docs/blog/2019-11-19-gatsby-brings-speed-to-mediacurrent-com/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Gatsby Brings Speed and Simplicity to Mediacurrent.com
+date: 2019-11-19
+author: Mark Casias
+excerpt: "We believe Gatsby is a superior solution that can work for many organizations looking to leverage a fast, secure React-based front-end for their website."
+tags:
+  - drupal
+  - case-studies
+---
+
+Back in 2016 we were planning the latest [Mediacurrent.com](https://www.mediacurrent.com) site redesign and were intrigued by the decoupled approach which was gaining popularity in the Drupal community. We liked the idea of breaking apart the front-end and backend so that we could have more flexibility in how we executed redesigns (i.e. redesigns would not be tied to the Drupal upgrade lifecycle). 
```

Comment:
Can we be consistent by either changing it to back-end or frontend?

Reply:
Which would you prefer? I have seen it both ways and even to throw a bigger wrench in there, we could separate it to back end and front end...

Reply:
I went ahead and switched them all to back-end and front-end since I had more of the dashed versions than not..

Reply:
Sounds good.